code/_onclick/observer.dm

```diff
@@ -8,7 +8,7 @@
 			return									// seems legit.
 
 	// Things you might plausibly want to follow
-	if((ismob(A) && A != src) || istype(A,/obj/machinery/bot) || istype(A,/obj/singularity))
+	if((ismob(A) && A != src) || istype(A,/obj/machinery/bot) || istype(A,/obj/singularity) || istype(A,/obj/mecha))
```

Comment:
holy shit this is horrible FUCK

remove this entire check

port ghost code from /tg/ where we can orbit any object/mob and has a Point of Interest system so you can add stuff to be followable on the fly